tests/test_sample_data.py

```diff
@@ -43,67 +49,57 @@ def test_other_input(make_files):
     with pytest.raises(ValueError):
         sample.sample_data(str(make_files.join('df.pkl')))
 
-def test_sample_record(make_files):
-    path_to_file = str(make_files.join('df.csv'))
-    df = pd.read_csv(path_to_file)
+def test_sample_record():
```

Comment:
Don't forget to add 1/2 line saying what the tests are doing!

Reply:
Made changes in new commit!